manifests/system.pp

```diff
@@ -76,8 +84,10 @@
   # the fact won't work until rvm is installed before puppet starts
   if $facts['rvm_version'] and !empty($facts['rvm_version']) {
     if ($version != undef) and ($version != present) and ($version != $facts['rvm_version']) {
-      if defined(Class['rvm::gnupg_key']) {
-        Class['rvm::gnupg_key'] -> Exec['system-rvm-get']
+      if $signing_keys {
```

Comment:
if we enforce the hash as datatype, we do not need this line. it is always true

Reply:
With the current module version, it's possible to set `gnupg_key_id` to `false` to disable key management. 
I was thinking about to drop this, but there might be valid reasons, to manage keys in a different way.

Reply:
In that case people could set `$singing_keys` to `{}`